### PR TITLE
Config test

### PR DESCRIPTION
**Pull request by JingTian2022**
I add some tests for the configuration file and add some more detailed instructions about how to set up the configuration file.

### CODE REVIEW

File: stock_getter/backend_stockgetter/test_config.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import unittest
+import configparser
+
+# Test if local configuration file meet the requirementsa and ready to use in app 
+config = configparser.ConfigParser()
+config.read('api_keys.ini')
```
Comment (SylvieXiang) [inline stock_getter/backend_stockgetter/test_config.py:6]: Just to check, this is a test for someone to make sure their created file is good? We no longer have this file anywhere in the repo.

File: stock_getter/backend_stockgetter/test_config.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import unittest
+import configparser
+
+# Test if local configuration file meet the requirementsa and ready to use in app
```
Comment (SylvieXiang) [inline stock_getter/backend_stockgetter/test_config.py]: meet -> meets
requirementsa -> requirements

File: stock_getter/backend_stockgetter/test_config.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+config.read('api_keys.ini')
+
+class TestConfigurationFile(unittest.TestCase):
+   
+    def test_file_exists(self):
+        # test if configuration file exists
+        self.assertTrue(config)
+
+    def test_section_name(self):
+        # test if configuration file contians correct section
```
Comment (SylvieXiang) [inline stock_getter/backend_stockgetter/test_config.py]: contains

File: stock_getter/backend_stockgetter/readme.txt [hunk 4]
```diff
@@ -1,15 +1,39 @@
...
+
+
+Note: In case your personal API URLs do not work or encounted any formating issues, you can use our provided URLs with correct format below:
+
+[API_URLS]
+stock_list = https://fcsapi.com/api-v3/stock/list?country=United-states&access_key=4wcNF3TiJ7XVx6p8zDt21
+stock_data_daily = https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&outputsize=full&apikey=Q5SUUT82ASKLSWB1
+stock_data_weekly = https://www.alphavantage.co/query?function=TIME_SERIES_WEEKLY&apikey=Q5SUUT82ASKLSWB1
+stock_data_monthly = https://www.alphavantage.co/query?function=TIME_SERIES_MONTHLY&apikey=Q5SUUT82ASKLSWB1
+
```
Comment (SylvieXiang) [inline stock_getter/backend_stockgetter/readme.txt:39]: We should probably add something here about testing if api_keys.ini is valid

File: stock_getter/backend_stockgetter/test_config.py (hunk 1 above)
Comment (JingTian2022) [reply stock_getter/backend_stockgetter/test_config.py:6]: Yep, this is only for users to test if their file is good

**Review by SylvieXiang — APPROVED** [review]

**Pull request merged**